src/gc.c

```diff
@@ -3253,8 +3251,18 @@ void jl_gc_init(void)
     uint64_t constrained_mem = uv_get_constrained_memory();
     if (constrained_mem > 0 && constrained_mem < total_mem)
         total_mem = constrained_mem;
-    max_total_memory = total_mem / 10 * 6;
+    uint64_t ngigs = total_mem / 10e9;
```

Comment:
/ (1 << 30)?

Reply:
Why not simply >> 30?
And isn't 10e9 not 10GB?

But as you are using double anyway later why not get rid of ngigs and do simply:

`if (total_mem < 128e9) {
      double percent = total_mem * 2.34375e-12 + 0.6;
`